rust/gsc/src/read_job.rs

```diff
@@ -18,14 +30,14 @@ pub async fn read_job(
 
     let result = conn
         .as_ref()
-        .transaction::<_, ReadJobResponse, DbErr>(|txn| {
+        .transaction::<_, (i32, ReadJobResponse), DbErr>(|txn| {
```

Comment:
This would become`.transaction::<_, (job::Entity, ReadJobResponse), DbErr>`

Reply:
Arc<job::Model> if we take my compromise above I think...actully even if we don't compromise it would need to be an Arc